reana_workflow_engine_snakemake/executor.py

```diff
@@ -154,55 +116,112 @@ def _get_container_image(self, job: Job) -> str:
             log.info(f"No environment specified, falling back to: {container_image}")
         return container_image
 
-    def _handle_job_status(self, job, status):
+    def _handle_job_status(
+        self, job: Job, job_status: JobStatus, workflow_status: RunStatus
+    ) -> None:
         workflow_uuid = os.getenv("workflow_uuid", "default")
         job_id = job.reana_job_id
-        log.info(f"{status} job: {job_id}")
+        log.info(f"{job_status} job: {job_id}")
```

Comment:
This prints something `JobStatus.finished job: af5b1b8d-6683-46ec-9fd9-f40b27e7b0b9` or `JobStatus.failed job: None`. Should we access `.name`?

Perhaps for the failed ones we can print the name instead, or always both id and name? It's not very helpful to get `None` there.

Reply:
Good catch. I aligned JobStatus here to match JobStatus enum in reana-db in case we will unify stuff later and forgot to update this comment

It should be `JobStatus.finished job: <job-id>` but I will change message to `f"{job.name} job is {job_status.name}. job id: {job_id}"`. It will look like "helloworld job is failed. job_id: af5b1b8d-6683-46ec-9fd9-f40b27e7b0b9" or "helloworld job is failed. job_id: None".

Reply:
Done